src/dotnet-core-uninstall/Shared/Commands/UninstallCommandExec.cs

```diff
@@ -201,25 +207,43 @@ private static IEnumerable<string> ParseCommandToArgs(string command)
             return args;
         }
 
-        private static void AskIt(IEnumerable<Bundle> bundles)
+        private static bool AskIt(IDictionary<Bundle, string> bundles)
```

Comment:
nit, good to improve but not blocking: the return value of AskIt is not obvious, it is the user's answer? or it is if this operation succeed or not.

A quick fix is to Rename it "AskItAndReturnUserAnswer" or simply add documentation.
A bigger design style is to make side effects(print to screen) API be void, and compute (query) only API to be return only and side effects free. So let it return a value might not be ideal
https://blog.ploeh.dk/2015/10/08/command-query-separation-when-queries-should-have-side-effects/